src/Chapter1.hs

```diff
@@ -568,7 +574,10 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' = True
```

Comment:
Nice one! 👍🏼 
As the or operator `||` returns `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u'
```